src/System.Private.CoreLib/shared/Interop/Windows/Kernel32/Interop.Globalization.cs

```diff
@@ -101,8 +101,8 @@ internal static unsafe partial class Kernel32
                     int cchStr);
 
 #if !ENABLE_WINRT
-        [DllImport("Kernel32.dll", CharSet = CharSet.Auto)]
-        internal static extern bool GetUserPreferredUILanguages(uint dwFlags, out uint pulNumLanguages, char[]? pwszLanguagesBuffer, ref uint pcchLanguagesBuffer);
+        [DllImport("kernel32.dll", CharSet = CharSet.Auto)]
+        internal static extern bool GetUserPreferredUILanguages(uint dwFlags, out uint pulNumLanguages, char* pwszLanguagesBuffer, ref uint pcchLanguagesBuffer);
```

Comment:
This looks like an incomplete fix. We are still going to get a IL stub for this one. Maybe split it into a separate change.

Reply:
I'll just fix the rest of the marshalling. The double call is so cheap as to be meaningless as it isn't repeated regularly so far as I can tell.